pkg/index/corpus.go

```diff
@@ -463,8 +463,8 @@ func (c *Corpus) scanFromStorage(s sorted.KeyValue) error {
 		return err
 	}
 
-	c.files = make(map[blob.Ref]camtypes.FileInfo, len(c.camBlobs["file"]))
-	c.permanodes = make(map[blob.Ref]*PermanodeMeta, len(c.camBlobs["permanode"]))
+	c.files = make(map[blob.Ref]camtypes.FileInfo, len(c.camBlobs[string(schema.FileCamliType)]))
```

Comment:
likewise: camBlobs could be keyed by the new type?

Reply:
Done